src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -991,6 +989,21 @@ private function getKernelRootHash(ContainerBuilder $container)
     }
 
     /**
+     * Gets the path of a given bundle using Bundle#getPath() when available.
+     *
+     * @param string $fqcn The bundle FQCN
+     *
+     * @return string
+     */
+    private function getBundlePath($fqcn)
+    {
+        $reflection = new \ReflectionClass($fqcn);
+        $getPathReflection = new \ReflectionMethod($fqcn, 'getPath');
+
+        return $getPathReflection->invoke(new $fqcn()) ?: dirname($reflection->getFilename());
```

Comment:
@stof Of course ... Do you know how a way to access the bundle instance directly?